opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -1786,36 +1786,42 @@ bool Well::cmp_structure(const Well& other) const {
         && (this->getHeadJ() == other.getHeadJ())
         && (this->hasRefDepth() == other.hasRefDepth())
         && (!this->hasRefDepth() || (this->getRefDepth() == other.getRefDepth()))
-        && (this->getPreferredPhase() == other.getPreferredPhase())
-        && (this->udq_undefined == other.udq_undefined)
-        && (this->getConnections() == other.getConnections())
         && (this->getDrainageRadius() == other.getDrainageRadius())
         && (this->getAllowCrossFlow() == other.getAllowCrossFlow())
         && (this->getAutomaticShutIn() == other.getAutomaticShutIn())
+        && (this->udq_undefined == other.udq_undefined)
+        && (this->getPreferredPhase() == other.getPreferredPhase()) // wellType()
         && (this->getEfficiencyFactor() == other.getEfficiencyFactor())
+        && (this->getConnections() == other.getConnections())
         ;
 }
 
 
 bool Well::operator==(const Well& data) const {
     return this->cmp_structure(data)
-        && (this->getSolventFraction() == data.getSolventFraction())
-        && (this->getEconLimits() == data.getEconLimits())
-        && (this->isProducer() == data.isProducer())
-        && (this->getFoamProperties() == data.getFoamProperties())
-        && (this->getStatus() == data.getStatus())
+        && (this->wpave_ref_depth == data.wpave_ref_depth)
+        && (this->gas_inflow == data.gas_inflow)
```

Comment:
having no idea whether this one should be added since there is some special comment with it, 

```
// Will NOT be loaded/assigned from restart file
```

Reply:
> since there is some special comment with it,

That "just" means that we currently don't store the gas inflow model anywhere in the restart file so if a run uses a non-default model (i.e., different from the standard one&ndash;`STD`), then a restarted run of that case won't reflect the inflow model.  We'll rectify this issue when needed.